src/CoreBundle/private/modules/CMSUpdateManager/CMSUpdateManager.class.php

```diff
@@ -59,11 +59,18 @@ public function GetHtmlHeadIncludes()
     {
         $aIncludes = parent::GetHtmlHeadIncludes();
         $aIncludes[] = '<link href="'.TGlobal::GetPathTheme().'/css/modules/updateManager.css" rel="stylesheet" type="text/css" />';
-        $aIncludes[] = '<script src="'.TGlobal::GetStaticURLToWebLib('/javascript/modules/updateManager/updateManager.js').'" type="text/javascript"></script>';
 
         return $aIncludes;
     }
 
+    public function GetHtmlFooterIncludes()
```

Comment:
Looks reasonable.

But is there maybe a special reason (here) why this change is needed?

Reply:
The CHAMELEON variable is now defined only in the footer and is required by the update manager, so the change is needed, yes.